controllers/posts.js

```diff
@@ -7,8 +7,8 @@ export const getAllPosts = async (req, res) => {
 };
 
 export const createPost = async (req, res) => {
-  const { body } = req;
-  const newPost = await (await Post.create({ ...body })).populate('author');
+  const { sanitizedBody } = req;
+  const newPost = await (await Post.create({ ...sanitizedBody })).populate('author');
```

Comment:
You don't need to spread this. Surely thats a residual from old logic where I added an additional prop or something 😅